Makefile.am

```diff
@@ -73,12 +73,19 @@ $(ALL_KEYRINGS):
 	keys='$^'; \
 	for key in $$keys; do \
 	  $(GPG) --homedir gnupg --no-default-keyring \
-	    --keyring $(builddir)/$@ --import $$key || exit 1; \
+	    --keyring $(builddir)/$@.tmp --import $$key || exit 1; \
 	done
+	$(GPG) --homedir gnupg --no-default-keyring \
+	  --keyring $(builddir)/$@.tmp --export > $(builddir)/$@
```

Comment:
No, `make` will use the exit status of the command like any other simple `make` command. The reason it's needed above is because it's in the middle of a compound command. The only way that one would fail is if the last command in the loop was a failure.